File: Oops Notes/NOTES FOR STUDIES/PROJECT MAKED/OurVBApp/OurVBApp/Form1.vb
Public Class Form1

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Dim ds As New DataSet
        Dim sql As String
        sql = "select * from userinfo where empid='" & TextBox1.Text & "' and password='" & TextBox2.Text & "'"
        ds = SelectQuery(sql)
        If (ds.Tables("a").Rows.Count > 0) Then
            MDIParent1.Show()
            Me.Hide()
        Else
            MessageBox.Show("Invalid ")

        End If
    End Sub

    Private Sub TextBox1_KeyPress(ByVal sender As System.Object, ByVal e As System.Windows.Forms.KeyPressEventArgs) Handles TextBox1.KeyPress
        If Asc(e.KeyChar) = Keys.Enter Then
            TextBox2.Focus()
        End If
    End Sub
End Class

File: Oops Notes/NOTES FOR STUDIES/New folder (2)/19 June 12 Data/PassByOut/PassByOut/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace PassByOut
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        //GD
        class Test
        {
            public void show(int x, int y, out int z)
            {
                z = x + y;
            }
        }
        private void button1_Click(object sender, EventArgs e)
        {
            int a = 10, b = 30, c=10;
            Test t = new Test();
            t.show(a, b, out c);
            MessageBox.Show("Total :"+c);

        }
    }
}
